items_orders_many_to_many_tables_recipe.md

```diff
@@ -0,0 +1,155 @@
+Two Tables (Many-to-Many) Design Recipe Template
```

Comment:
I went about this as one-to-many (and got confused). I like how you have gone with a more logical approach as the one-to-many tables structure doesn't quite fit this brief.